aws/resource_aws_codebuild_project.go

```diff
@@ -180,6 +180,16 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 				Default:      "60",
 				ValidateFunc: validateAwsCodeBuildTimeout,
 			},
+			"badge_enabled": {
```

Comment:
In order for Terraform to determine that this parameter has changed from the API, we need to call `d.Set("badge_enabled", project.Badge.BadgeEnabled)` in the read function.